environments/local/group_vars/all

```diff
@@ -0,0 +1,44 @@
+nginx:
+  conf:
+    dir: /tmp/nginx
+whisk:
+  version:
+    date: "21-02-2016"
+    name: "ansible"
+  ssl:
+    cert: config/keys/openwhisk-self-cert.pem
+    key: config/keys/openwhisk-self-key.pem
+  logs:
+    dir: /tmp/wsklogs
+docker:
+  restart:
+    policy: always
+
+controller:
+  host: 172.17.0.1
+  port: 10001
```

Comment:
We should try to get rid of `host` as it duplicates properties from the inventory.